dashboard/app/views/levels/editors/_all.html.haml

```diff
@@ -35,44 +50,45 @@
   #audit_log_info.collapse
     = render partial: 'levels/editors/fields/audit_log', locals: {f: f}
 
-.field
```

Comment:
Everything from here down in this file is all just an indent change besides new line 53 which is the thing everything was indented inside of.